packages/app-mobile/components/screens/SearchScreen/index.tsx

```diff
@@ -46,13 +47,19 @@ const useStyles = (theme: ThemeStyle) => {
 				paddingRight: theme.marginRight,
 				backgroundColor: theme.backgroundColor,
 			},
+			rootStyle: {
+				...theme.rootStyle,
+
+				// TODO: Improve. This hides the component without unmounting it.
+				flex: !visible ? 0.001 : 1,
```

Comment:
See the similar logic in `Notes.tsx`:
https://github.com/laurent22/joplin/blob/90640e590e3d82762f6f5ffa93c02e46d61664dd/packages/app-mobile/components/screens/Notes.tsx#L243
...and the equivalent logic in `search.tsx` prior to the refactoring:
https://github.com/laurent22/joplin/blob/f772cc500c7b19b2870fa1634d54a5a275a73323/packages/app-mobile/components/screens/search.tsx#L178